components/board/pi/common/errors.go

```diff
@@ -0,0 +1,151 @@
+package picommon
+
+// PiGPIOErrorMap maps the error codes to the human readable error names. This can be found at the pigpio C interface.
+var PiGPIOErrorMap = map[int]string{
```

Comment:
can this be accessed directly from the library rather than copying and pasting all codes?
only if this isn't too hard since we already have use Cgo for the library. I know other libraries copy this map just like you did.

Reply:
updated

Reply:
that's not what I meant... we shouldn't have a map of error code -> error messages in our code if we can get it from the library.

Reply:
It looks like they are just variable definitions in the library - this gets the corresponding error code in the case that they are changed. The descriptions are `error_code_name: comments from the docs page`. From what I can tell, I don't think a map already exists